api/types/src/transaction.rs

```diff
@@ -55,6 +55,7 @@ pub struct TransactionOnChainData {
     pub info: aptos_types::transaction::TransactionInfo,
     pub events: Vec<ContractEvent>,
     pub accumulator_root_hash: aptos_crypto::HashValue,
+    pub changes: aptos_types::write_set::WriteSet,
```

Comment:
nit: how about renaming this to `write_set` to be consistent with the naming we are using in other parts of the code.

Reply:
`changes` is already used in the `DirectWriteSet` serialization